docs/examples/rrv.ipynb

```diff
@@ -4,22 +4,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Respiratory Rate Variability (RRV)"
+    "# Respiratory Rate Variability Analysis"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Respiratory Rate Variability (RRV), or variations in respiratory rhythm, are crucial indices of general health and respiratory complications. This example shows how to use NeuroKit to perform RRV analysis.\n",
-    "\n",
-    "\n",
-    "**I don't have any respiration data :'(**: No problem, see how you can extract **respiration from cardiac activity** (e.g., ECG) in [this example](https://github.com/neuropsychology/NeuroKit/issues/87)  [TODO]"
```

Comment:
You removed this added line (maybe you didn't update the latest version from dev)

Reply:
Still working on my Git workflow. Sorry for that